.github/ISSUE_TEMPLATE/bugs.md

```diff
@@ -0,0 +1,30 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Current Behavior
+<!--- Tell us what happens instead of the expected behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
```

Comment:
a solution isn't the same thing as a cause. The comment is asking for a "fix/reason", which can be two separate things. But the header in line 9 is asking for a possible "solution"